Exercise02-Basics/Svetlin-Trayanski/BASICS01-002.js

```diff
@@ -0,0 +1,12 @@
+//BASICS01-002: Imagine, you are solving a math problem. Declare ten variables, which you might need in your program.
+
+let x = 5
+let y = 6
+let h = 7
+let geometricalshape = String(input[0])
+let squareArea = a*a
+let triangleArea = 0.5 * b * h
+let rectangleArea = a * b
+let p = Math.PI
```

Comment:
`let geometricalshape = String(input[0])`
in our case we don't have any input date you modify that to a string value

Line 7 is correct, but following the convention would be `let squareArea = a * a` 

`let p = Math.PI` --> try to use a value instead of the `Math` library